### DISCUSSION TITLE
Error while running Haar cascade face detection in  Axis P1467-LE camera

### DISCUSSION THREAD

**Discussion by RaySayantan99** [AxisCommunications/acap-computer-vision-sdk-examples#165] [General] [answered]
Hi,
I tried to run face detection application using Haar cascade algorithm on Axis P1467-LE camera .

I am getting the following error :

<img width="659" alt="image" src="https://github.com/AxisCommunications/acap-computer-vision-sdk-examples/assets/141122487/88bc1f5b-8d12-4c91-b0c5-4eb64f1331f8">

this is the piece of code for Haar Cascade Face detection:

<img width="491" alt="image" src="https://github.com/AxisCommunications/acap-computer-vision-sdk-examples/assets/141122487/156fd4d9-e17c-499a-af7c-2bcc92813aff">

**Comment by shreyasatwork** [comment, marked as answer]
Hello @RaySayantan99,

This seems more like a specific file error, can you make sure:

- The file path that you provided is correct and file exists in that path (maybe try using absolute path).
- Ensure you have necessary permissions to read the XML file.
- Ensure the file is not corrupted and it is well-formed with correct structure for a HAAR cascade.

**Reply by RaySayantan99** [reply]
Hi @shreyasatwork , 
Thanks for listing out the possible reasons . I downloaded the data folder from haar cascade github repo and placed it inside the app folder of the project. It worked after that.
